.codacy.yml

```diff
@@ -0,0 +1,5 @@
+---
+exclude_paths:
+  - 'opt4j-viewer/src/main/java/ptolemy/**/*'
```

Comment:
What is your gut feeling. Should we also exclude the unit test or are the changes you did also senseful for them?

Reply:
I reworked the tests, so I would keep it currently. Let us discuss this as part of issue #39.

Reply:
fine for me.